mobile/src/bundles/talent/components/profile-form/profile-form.tsx

```diff
@@ -1,37 +1,78 @@
+import { type NavigationProp } from '@react-navigation/native';
 import React from 'react';
 
 import {
     Button,
-    Checkbox,
     FormField,
     Input,
     ScrollView,
     Selector,
     Slider,
     View,
 } from '~/bundles/common/components/components';
-import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
+import {
+    TalentOnboardingScreenName,
+    TalentOnboardingStepState,
+} from '~/bundles/common/enums/enums';
+import {
+    useAppForm,
+    useCallback,
+    useNavigation,
+} from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
-
+import { type TalentOnboardingNavigationParameterList } from '~/bundles/common/types/types';
 import {
-    CURRENT_LOCATION_OPTIONS,
-    JOB_TITLE_OPTIONS,
-    TALENT_PROFILE_DEFAULT_VALUES,
-} from './constants/constants';
+    CountryList,
+    EmploymentType,
+    JobTitle,
+} from '~/bundles/talent/enums/enums';
+import { type ProfileStepDto } from '~/bundles/talent/types/types';
+import { ProfileStepValidationSchema } from '~/bundles/talent/validation-schemas/validation-schemas';
+
+import { EmploymentTypes } from './employment-types';
 import { styles } from './styles';
 
+const jobTitleOptions = Object.values(JobTitle).map((title) => ({
+    value: title,
+    label: title,
+}));
+
+const locationOptions = Object.values(CountryList).map((country) => ({
+    value: country,
+    label: country,
+}));
+
+const employmentTypeOptions = Object.values(EmploymentType).map((type) => ({
+    value: type,
+    label: type,
+}));
```

Comment:
let's refactor this, let's create only values[] for example since value and label are the same

Reply:
done